lib/resources/apache_conf.rb

```diff
@@ -21,7 +21,7 @@ class ApacheConf < Inspec.resource(1)
 
     def initialize(conf_path = nil)
       @conf_path = conf_path || inspec.apache.conf_path
-      @conf_dir = File.dirname(@conf_path)
+      @conf_dir = inspec.os.debian? ? File.dirname(@conf_path) : inspec.apache.conf_dir
```

Comment:
@jeremymv2 I am thinking about moving that logic to `inspec.apache`. So we would use `File.dirname(@conf_path)` only if `conf_path` is defined. Otherwise we always read the `conf_dir` from the apache resource. What do you think?